velox/dwio/dwrf/reader/BinaryStreamReader.cpp

```diff
@@ -111,7 +111,8 @@ BinaryStreamReader::getStatistics() const {
         "Corrupted file detected, Footer stats are missing, but stripes are present");
     for (auto node = 0; node < typesSize; node++) {
       if (columnSelector_.shouldReadNode(node)) {
-        stats[node] = proto::ColumnStatistics();
+        const proto::ColumnStatistics cs = proto::ColumnStatistics();
```

Comment:
@Yuhta Thank you for your review, It should be that temporary local variables are used here, I will modify it.